run-integrated-tiger.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
```

Comment:
This file should be 

- moved to `common/scripts/ci` . and file name should be s.t like `run-sdk-backend-tiger-integration-tests-with-live-backend.sh`
- Or moved to `libs/sdk-backend-tiger/scripts` and replace `integrated-test-AIO.sh`